lms/routes.py

```diff
@@ -148,7 +148,10 @@ def includeme(config):  # pylint:disable=too-many-statements
     config.add_route("admin.instance.create", "/admin/instance/create")
     config.add_route("admin.instance.upgrade", "/admin/instance/upgrade")
     config.add_route("admin.instance", "/admin/instance/{id_}/")
+    config.add_route("admin.instance.info", "/admin/instance/{id_}/info")
```

Comment:
The "info" tab is the default, so it doesn't need a separate URL from `/admin/instance/{id_}`.